pg_metadata/dock.py

```diff
@@ -66,12 +77,81 @@ def __init__(self, parent=None):
         menu.addAction(self.auto_open_dock_action)
         self.config.setMenu(menu)
 
+        # Setting PDF/HTML menu
+        self.save_button.setAutoRaise(True)
+        self.save_button.setToolTip(tr("Save metadata"))
+        self.save_button.setPopupMode(QToolButton.InstantPopup)
+        self.save_button.setIcon(QIcon(QgsApplication.iconPath('mActionFileSave.svg')))
+
+        self.save_as_pdf = QAction(
+            tr('Save content as PDF file'),
```

Comment:
```suggestion
            tr('Save as PDF…'),
```
UX, tip, ellipsis is used when there is another action needed after clicking a button.